### PR TITLE
Fix(Core/Script) Quest 12019: Last Rites

### PR DESCRIPTION
**Pull request by malcrom**
Quest is now working properly.

## Changes Proposed:
-  Replace SAI script with C++ script.

## Tests Performed:
- Builds
- Works minor visual issue with Blood Presence aura on some NPCs due to core condition handling. Does not affect quest.

## How to Test the Changes:
.add q 12019
.go c 101303
Talk to Thassarian

### CODE REVIEW

**Comment by malcrom** [comment]
Complete

**Comment by tkn963** [comment]
Compiling as we speak, will let you know my results when I'm done.

I'm writing this comment ahead of time only to say that you should probably mention that the quest is only for Alliance.

**Comment by tkn963** [comment]
Works fine 👍

File: data/sql/updates/pending_db_world/rev_1634939032065262806.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1634939032065262806');
+
+-- *** Quest "Last rites" ***
+
+-- Remove Quest SAI
+UPDATE `creature_template` SET `AIName`='' WHERE `entry` IN (26170,26203,25301,25250,25251);
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (26170,26203,25301,25250,25251);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1634939032065262806.sql:7]: @malcrom we should add `source_type` here to avoid deleting unwanted rows

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

**Comment by FrancescoBorzi** [comment]
thanks @tkn963 for testing this!

@malcrom I left a small comment, please check

File: data/sql/updates/pending_db_world/rev_1634939032065262806.sql (hunk 1 above)
Comment (malcrom) [reply data/sql/updates/pending_db_world/rev_1634939032065262806.sql:7]: Will update

Comment (malcrom) [reply data/sql/updates/pending_db_world/rev_1634939032065262806.sql]: Fixed

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**